src/api/menu/components/Menu.vue

```diff
@@ -3,39 +3,52 @@
     class="c-menu"
     :class="options.menuClass"
 >
-    <ul v-if="options.actions.length && options.actions[0].length">
+    <ul
+        v-if="options.actions.length && options.actions[0].length"
+        role="menu"
+    >
         <template
             v-for="(actionGroups, index) in options.actions"
         >
-            <li
-                v-for="action in actionGroups"
-                :key="action.name"
-                :class="[action.cssClass, action.isDisabled ? 'disabled' : '']"
-                :title="action.description"
-                :data-testid="action.testId || false"
-                @click="action.onItemClicked"
-            >
-                {{ action.name }}
-            </li>
             <div
-                v-if="index !== options.actions.length - 1"
-                :key="index"
-                class="c-menu__section-separator"
-            >
-            </div>
-            <li
-                v-if="actionGroups.length === 0"
                 :key="index"
+                role="group"
             >
-                No actions defined.
-            </li>
-        </template>
+                <li
+                    v-for="action in actionGroups"
+                    :key="action.name"
+                    role="menuitem"
+                    :class="[action.cssClass, action.isDisabled ? 'disabled' : '']"
```

Comment:
i don't know enough about this change to review this. You should quickly ping a css wiz